File: objective-c/StoreKit/StoreKit_v2/StoreKit_v2/ViewControllers/ARootNavigationController.m
//
//  ARootNavigationController.m
//  StoreKit_v2
//
//  Created by Alexandr Kirilov on 15/12/2022.
//

#import "ARootNavigationController.h"

@interface ARootNavigationController ()

@end

@implementation ARootNavigationController

-(instancetype) init {
	
	self = [super init];
	if (self) {
		
	}
	return self;
}

-(void) loadView {
    
    [super loadView];
    [self.view setBackgroundColor:__COLOR_BLUE];
	
	[self.navigationBar setBarStyle:UIBarStyleBlack];
}

-(void)viewDidLoad {
    
    [super viewDidLoad];
}

@end

File: swift/StoreKit/StoreKit2_v1/StoreKit2_v1/ViewControllers/ARootNavigationController.swift
//
//  ARootNavigationController.swift
//  StoreKit2_v1
//
//  Created by Alexandr Kirilov on 19/12/2022.
//

import UIKit

class ARootNavigationController: UINavigationController {


	override func loadView() {
		
		super.loadView();
		self.view.backgroundColor = __COLOR_GREEN_LIGHT;
		self.navigationBar.barStyle = .black;
	}
}
